desktop/apps/rss/templates/tracking.jade

```diff
@@ -1,21 +0,0 @@
-figure.op-tracker
-  iframe(hidden)
-    script( type="text/javascript" ).
-      PARSELY = { autotrack: false, onload: function() { PARSELY.beacon.trackPageView({ urlref: 'http://instantarticles.fb.com' }); return true; } }
-      !function(){var analytics=window.analytics=window.analytics||[];if(!analytics.initialize)if(analytics.invoked)window.console&&console.error&&console.error("Segment snippet included twice.");else{analytics.invoked=!0;analytics.methods=["trackSubmit","trackClick","trackLink","trackForm","pageview","identify","reset","group","track","ready","alias","page","once","off","on"];analytics.factory=function(t){return function(){var e=Array.prototype.slice.call(arguments);e.unshift(t);analytics.push(e);return analytics}};for(var t=0;t<analytics.methods.length;t++){var e=analytics.methods[t];analytics[e]=analytics.factory(e)}analytics.load=function(t){var e=document.createElement("script");e.type="text/javascript";e.async=!0;e.src=("https:"===document.location.protocol?"https://":"http://")+"cdn.segment.com/analytics.js/v1/"+t+"/analytics.min.js";var n=document.getElementsByTagName("script")[0];n.parentNode.insertBefore(e,n)};analytics.SNIPPET_VERSION="3.1.0";
-      analytics.load("#{sd.SEGMENT_WRITE_KEY_MICROGRAVITY}");
```

Comment:
Question here about sd/env..I added `SEGMENT_WRITE_KEY_MICROGRAVITY` specifically for this use case before the merge. I should be able to remove all references to it in this PR since the app doesn't use it anywhere else. I haven't done that yet though because I wasn't sure if we should update the /mobile folder with `SEGMENT_WRITE_KEY_MICROGRAVITY` instead of [SEGMENT_WRITE_KEY](https://github.com/artsy/force/blob/master/mobile/config.coffee#L43). Where does the separation of the Force vs MG keys happen when they have the same name?